server/hotrod/src/main/java/org/infinispan/server/hotrod/Authentication.java

```diff
@@ -124,7 +125,10 @@ private SaslServer createSaslServer(String mech) throws Throwable {
       SslHandler sslHandler = channel.pipeline().get(SslHandler.class);
       if (sslHandler != null) {
          try {
-            principals.add(sslHandler.engine().getSession().getPeerPrincipal());
+            // Principal peerPrincipal = sslHandler.engine().getSession().getPeerPrincipal();
+            // Workaround WFSSL-19
```

Comment:
It looks like WFSSL-19 was fixed in 1.0.9.Final. 
Is the workaround still required?

Reply:
I can remove the workaround